optimize-nextflow.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+
+import json
+import shutil
```

Comment:
Nits: pep8 imports, docstrings, etc.

Reply:
What do you mean by PEP8 imports? Isn't this already compliant with PEP8? 

I'll add some docstrings, although I figured that the functions were already self-explanatory based on their names.

Reply:
Ah yes, sorry about that. I forgot that there isn't actually a pep8 rule for sorting import statements (alphabetically), but even if there was, you do actually sort them first by `import` statements then by the `from... import` statements...

Re: docstrings, that was more nit.  I agree most, if not all, are self explanatory.